starter-code/styles/base.css

```diff
@@ -1,5 +1,7 @@
 /* TODO: Add a rule for images, so they never exceed their container's width. */
-
+img {
```

Comment:
Using width at 100% causes the images to become 100% of the width of the screen. But the TODO specified that the images should not exceed their containers width, not the screen width. How could we fix this?
Let me know if you have questions.